exporter/metric/metric.go

```diff
@@ -419,17 +419,11 @@ func countToTypeValueAndTimestamp(count aggregator.Count, kind apimetric.NumberK
 				DoubleValue: float64(value),
 			},
 		}, t, nil
-	case apimetric.Uint64NumberKind:
```

Comment:
Note to myself: support for Uint64 was removed in https://github.com/open-telemetry/opentelemetry-go/issues/851